CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Fixes
+
+- Map iOS native events - OnCrashedLastRun and BeforeSend ([#2102](https://github.com/getsentry/sentry-dotnet/pull/3958))
```

Comment:
@aritchie this is what we did to implement this feature. What does it mean for the user though? Why should they care?

This isn't necessarily a fix either. It's just new functionality, AFAIK. I'd probably put under `Features`.